content/en/docs/ops/ambient/install/helm-installation/index.md

```diff
@@ -147,18 +147,14 @@ installed above.
     {{< text syntax=bash snip_id=delete_base >}}
     $ helm delete istio-base -n istio-system
     {{< /text >}}
+1. Delete CRDs Installed by Istio (Optional)
+
+{{< text syntax=bash >}}
```

Comment:
i think i had not aligned it properly :D

Reply:
Fixed, PTAL again, thanks!